mne/annotations.py

```diff
@@ -598,6 +598,7 @@ def to_data_frame(self, time_format="datetime"):
         ----------
         %(time_format_df_raw)s
 
+        Default is ``datetime``.
```

Comment:
probably need to indent the new text, and maybe also nix the newline / move the newline to before the versionadded directive